src/ripple/shamap/SHAMapItem.h

```diff
@@ -40,6 +40,7 @@ class SHAMapItem
 public:
     SHAMapItem (uint256 const& tag, Blob const & data);
     SHAMapItem (uint256 const& tag, Serializer const& s);
+    SHAMapItem (uint256 const& tag, Serializer&& s);
```

Comment:
In this case, `Serializer` is fast to move from.We could move from a value type.